### ISSUE TITLE
How do I use this repository

### ISSUE THREAD

**Issue by erelsgl** [education/autograding-example-cpp#2]
Sorry for the newbie question.. I am trying to make the first exercise in my Github Classroom course. I got here from the "**Add autograding tests**" tab in the exercise settings. What exactly do I do with this repository? Should I construct a similar repository, with all the tests needed for the auto-grading, and use it as the answer to the tab "**Add a template repository to give students starter code**" tab? 
If not, then where exactly should I put the automatic tests that I write?

**Comment by erelsgl** [comment]
An additional question is: how is the grade calculated, from the results of the autograding? Is it just the percent of passing tests?

**Comment by MuhammadMoustafa** [comment]
Yes exactly, you need to provide a template folder when you create an assignment.
or ask each student to add these files, but this is unneeded hassle.
creating a template folder will give you the flexibility to add stub functions and ask your students ti fill it so they don't mess up the repo.
 
The bottom line here is that the files used by -Run command- exists.

For the second question:
when you add a test, There is a text box with label Points.
If you need grade each test separately, you may add multiple test files, 
but make sure that you write the right command in "Run command" Text box.
